### PR TITLE
Add a newspost for playtest-20200329

### PR DESCRIPTION
**Pull request by abcdefg30**
Open:
- [x] `GITHUB_PLAYTEST_ID`
- [x] Changelog
- [x] Date and time

### CODE REVIEW

File: content/news/playtest-20200323.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Today's playtest constitutes our last release candidate for this cycle and adresses the issues discovered in the previous playtest.
```
Comment (pchote) [inline content/news/playtest-20200323.md]: I'm always hesitant to use such definitive wording - can we add a "hopefully" before or after the constitutes?

File: content/news/playtest-20200323.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The [changes in playtest-20200323](TODO) include:
+
+* The return of the ability to use right-click orders and scrolling at the same time
+* A fix for aircraft not taking off when no rallypoint is set
+* Improved pathfinding for all units, especially infantry
```
Comment (pchote) [inline content/news/playtest-20200323.md]: Only infantry? #17777 was purely a performance fix.

**Comment by pchote** [comment]
I wonder whether it is worthwhile (and not tactless) to mention the recent surge in player count, or the virus situation more generally.

File: content/news/playtest-20200323.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Today's playtest constitutes our last release candidate for this cycle and adresses the issues discovered in the previous playtest.
+
+The [changes in playtest-20200323](TODO) include:
```
Comment (pchote) [inline content/news/playtest-20200323.md]: We could mention the weather effects fix too

**Comment by pchote** [comment]
We should also mention the new mod sdk release, which I will write release notes for and tag after we have an engine playtest tag.

File: content/news/playtest-20200323.md (hunk 3 above)
Comment (abcdefg30) [reply content/news/playtest-20200323.md]: I thought the weather effects PR was a regression fix?

Comment (pchote) [reply content/news/playtest-20200323.md]: yeah, i meant to include a mention for regression fixes like this.

**Comment by abcdefg30** [comment]
Updated.

**Pull request merged**